src/chilean-chickpeas.md

```diff
@@ -0,0 +1,31 @@
+# Chilean Chickpeas
+
+Traditional chilean dish.
+- ⏲️ Prep / Cook time: 30 - 40 m
+- 🍽️ Servings: 6
+
+## Ingredients
+
+- Dry chickpeas: 500 g
+- Sweet potato squash: 250 g
+- Onion: 1
+- Carrot: 1 cup
+- Pepper: 1 cup
```

Comment:
```suggestion
- Grated carrot: 1 cup
- Grated pepper: 1 cup
```

Specifying that the carrot and pepper is grated right away makes it more clear on how you measure a carrot by volume.

Also, what type of pepper is this?